### ISSUE TITLE
streaming OSC data

### ISSUE THREAD

**Issue by fcolino** [sccn/labstreaminglayer#60]
Hello, 

I am new to LSL and I would like to receive data from an open sound control (OSC) data source. Additionally, I'd like to record and monitor "live" data. The OSC data source is a MUSE headband. I'm running Windows 10 (64-bit) and Matlab 2015b. I've already downloaded an installed the LSL files and dependencies. Any guidance would be appreciated. 

Thank you.

**Comment by sccn** [comment]
Hi, fcolino. I can point you to some work that has already been done with OSC and muse (please send me a personal email at dmedine@ucsd.edu, and I will track this stuff down for you), but so far the results are not great. There is a bug in the muse code for LSL support, and it seems that converting the OSC timestamps is tricky business. This reminds me that I should ping muse again to see if they have fixed the bug yet (they promised they would!).

**Comment by sccn** [comment]
Sorry, moving this back to the forum level. So, yes, it looks the arrival of working LSL on muse is indefinite for the time being, however, Olav Krigolson has mentioned that he is working on a project called Engage that should have solutions to the muse+LSL problem, and that its release is imminent!

**Comment by chkothe** [comment]
Yeah, I'd be interested in what the problem is on there. Getting the
timestamps right is as simple as not overriding them in the first place.

Christian
